api/src/models/student.js

```diff
@@ -3,6 +3,7 @@ const uniqueValidator = require("mongoose-unique-validator");
 /**
  * @swagger
  *  components:
+ *   $ref: './course.js/#components'
```

Comment:
Uh dont know how to comment on unchanged code but is the todo at the bottom of this file supposed to be left like that?

Reply:
i will change this